LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We should dual license this like all of our other software

ref https://github.com/astral-sh/uv#license